recipe/meta.yaml

```
@@ -145,6 +144,7 @@ outputs:
 
 about:
   home: https://www.gnu.org/software/binutils/
+  dev_url: https://sourceware.org/git/gitweb.cgi?p=binutils-gdb.git
```

Isn't it technically

```suggestion
  dev_url: git://sourceware.org/git/gitweb.cgi?p=binutils-gdb.git
```

because if I `git clone https://sourceware.org/git/gitweb.cgi?p=binutils-gdb.git` I get a repository not found error.

Of course, the linter/conda build might not like `git://` URLs and `dev_url` might be meant to indicate something other than the repository...

---

@ifitchet The dev_url isn't necessarily the source URL. It's just a URL that points to where the development happens.